hack/test-go.sh

```diff
@@ -203,10 +206,13 @@ reportCoverageToCoveralls() {
 
 # Convert the CSV to an array of API versions to test
 IFS=',' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
+ETCD_PREFIX=${ETCD_STANDARD_PREFIX}
 for apiVersion in "${apiVersions[@]}"; do
   echo "Running tests for APIVersion: $apiVersion"
-  KUBE_API_VERSION="${apiVersion}" runTests "$@"
+  KUBE_API_VERSION="${apiVersion}" ETCD_PREFIX=${ETCD_STANDARD_PREFIX} runTests "$@"
```

Comment:
Locally, unit tests are now being run 3 times!
For v1beta1, beta3 and etcdPrefix.
@bgrant0607 @lavalamp 
Is this intended?

Maybe we can run with v1beta1 and standard prefix once and then v1beta3 and custom prefix (or vice-versa)?
Or if the test passes with a custom prefix, then it means that it will pass with the standard prefix as well?